resources/guidelines/testing/store/quality-guidelines-apps/README.md

```diff
@@ -142,6 +142,27 @@ Iframes, external scripts, or tracking pixels are not allowed in the description
 
 * The price and shopping cart button may not be covered by customizations - for example, "badges". Furthermore, the shopping cart button must always be clickable.
 
+
+### Dummy App 
+
+With a dummy app you can offer your middleware in the store, if you only use the APIs of Shopware. 
```

Comment:
I think `API service` is a more fitting description than middleware in this context.